README.md

```diff
@@ -70,6 +70,7 @@ Additionally,
 * _Edit_ a revision by pressing `e`
 * Git _push_/_fetch_ by pressing `g`
 * Undo the last change by pressing `u`
+* Redo the last change by pressing `R`
```

Comment:
Thanks! This also needed to change but I can do it after the merge.

Reply:
oh sorry that i missed it, thanks for fixing!